<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QWidget" name="EditorWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="SnakeList">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>75</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SnakeAddButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SnakeDelButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QListView" name="ObjectList">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>245</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ObjectAddButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>349</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ObjectDelButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>349</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QListView" name="ltemList">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>404</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ItemAddButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>520</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ItemDelButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>520</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>384</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Items</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>224</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Objects</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>56</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Snakes</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="gameField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>531</width>
     <height>491</height>
    </rect>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
